hackerrank/a-very-big-sum/challenge.js

```diff
@@ -0,0 +1,13 @@
+// Solve the challenge in this JavaScript file
+
+const aVeryBigSum = (ar) => {
+  let sum = 0;
+
+  ar.forEach((element) => {
+    sum += element;
+  });
+
+  return sum;
+};
```

Comment:
Use this opportunity to learn about `array.reduce`, it would be nice to use that instead.

see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce